backend/src/Controller/taskcontroller.ts

```diff
@@ -0,0 +1,58 @@
+import { Request, Response } from 'express';
+import Task from '../entities/task';
+import AppDataSource from "../dataBase/config";
+
+const addTask = async (req: Request, res: Response) => {
+  const { task, id } = req.body;
+  try {
+    if (!task) return res.status(400).send('Please enter the task');
```

Comment:
for these things we use validators, please learn about them and try to implement it.